src/main/java/com/theelixrlabs/healthcare/validation/Validator.java

```diff
@@ -58,4 +61,39 @@ public void validatePatientDTO(PatientDTO patientDTO) throws CustomException {
             throw new CustomException(messageUtil.getMessage(PatientConstants.INVALID_LAST_NAME_KEY));
         }
     }
+
+    /**
+     * Validating method to validate based on incoming request.
+     *
+     * @param doctorDto Data transfer object containing doctor information.
+     * @throws CustomException If validation fails or doctor already exists .
+     */
+    public void validateDoctor(DoctorDto doctorDto) throws CustomException {
+        if (doctorDto.getFirstName() != null) {
+            if (doctorDto.getFirstName().isEmpty()) {
+                throw new CustomException(messageUtil.getMessage(MessageConstants.DOCTOR_FIRST_NAME_SHOULD_NOT_BE_EMPTY));
+            } else if (!doctorDto.getFirstName().matches(DoctorConstants.CHARACTER_ONLY_REGEX_PATTERN)) {
+                throw new CustomException(messageUtil.getMessage(MessageConstants.DOCTOR_INVALID_FIRSTNAME));
+            }
```

Comment:
Why else if (...) ?
Can't we make just if (..) ?

also applies for all the else if condition below

Reply:
removed else put if part only